test/tests/functional/pbs_hook_modifyvnode_state_changes.py

```diff
@@ -292,6 +292,9 @@ def test_hook_state_changes_00(self):
         Test: induce a variety of vnode state changes with debug turned on
         and inspect the pbs log for expected entries
         """
+        if os.getuid() != 0 or sys.platform in ('cygwin', 'win32'):
+            self.skipTest("Test need to run as root")
```

Comment:
Is skip for cygwin and win32 required?

Reply:
I think the issue goes further than this test.  The test is failing in self.server.submit().  Shouldn't we be able to submit a reservation as ROOT_USER if not running as root?  Isn't the purpose of the user argument to run as someone else?  I think this fix should be in submit(), not this test.

Please file a ticket for submit().  For the time being, we can skip this test until we fix submit().